api/env.docker

```diff
@@ -19,5 +19,3 @@ SEMANTIC_VERSION=1.0.0
 ELASTICSEARCH_URL=es
 
 WATERMARK_ENABLED=True
-
```

Comment:
Why do we remove variable this from here?

Reply:
We should strive to remove this file entirely. I was confused at first when my local API wasn't filtering dead links given I had the variable commented in my `.env` file, then I found this was the problem. These are environment settings so it's not a good practice to track them in vcs either.

Reply:
Now that `env.template` isn't used for production (was it ever?) and neither is `env.docker`, and #2180 is implemented, I don't think these `env.docker` files make sense at all, really. We have appropriate defaults in `env.template` for local development, which is the only place that file is ever used, so this file feels duplicative indeed.

Reply:
I opened an issue to track this and fix it generally: https://github.com/WordPress/openverse/issues/2437